### PR TITLE
Update sample code of voice synthesis APIs.

### PR DESCRIPTION
**Pull request by hepower**
Update sample code of voice synthesis APIs to version V3.0 beta1

### CODE REVIEW

File: CustomVoice-API/CustomVoice-API/TestData/en-US_small.txt [hunk 1]
```diff
@@ -1,4 +1,50 @@
-But the gold standard gets a bad rap in textbooks as an archaic theory.
-Ron Paul said Tuesday he runs for president to show people the trouble America is in.
-I wish there was a show like that about personal finance.
-There's gotta be a little romance and good sex in the equation.
\ No newline at end of file
+﻿This is the waistline , and it's falling .
```
Comment (boltomli) [inline CustomVoice-API/CustomVoice-API/TestData/en-US_small.txt]: Is this change necessary?

Comment (hepower) [reply CustomVoice-API/CustomVoice-API/TestData/en-US_small.txt]: Yes. the new API has a limitation that at least the input text file shall contain 50 lines of inputs.
